node/npm_dependencies.ts

```diff
@@ -81,6 +81,10 @@ const safelyDetectTypes = async (packageJsonPath: string): Promise<string | unde
 // Workaround for https://github.com/evanw/esbuild/issues/1921.
 const banner = {
   js: `
+  import process from "node:process";
+  import { setImmediate, clearImmediate } from "node:timers";
```

Comment:
nit: Can we make the formatting match the imports we already had? No preference on which convention we pick, but let's try to be consistent.

Reply:
done in https://github.com/netlify/edge-bundler/pull/519/commits/0e45e3b23df5aad5fd0d3d9fabf8f5cd830497f8